client/app/hosting/hosting.constant.js

```diff
@@ -1,11 +1,24 @@
-angular.module('App').constant('HOSTING', {
-  cloudWeb: {
-    configurationQuota: {
-      unit: 'GB',
-      value: 15,
+angular
+  .module('App')
+  .constant('HOSTING', {
+    cloudWeb: {
+      configurationQuota: {
+        unit: 'GB',
+        value: 15,
+      },
     },
-  },
-  offers: {
-    START_10_M: 'START_10_M',
-  },
-});
+    offers: {
+      START_10_M: 'START_10_M',
+    },
+  })
+  .constant('HOSTING_RUNTIME', {
+    NODE: 'nodejs',
+    PHP: 'phpfpm',
+  })
+  .constant('HOSTING_STATUS', {
+    CREATED: 'created',
+    CREATING: 'creating',
+    DELETING: 'deleting',
+    REGENERATING: 'regenerating',
+    UPDATING: 'updating',
+  });
```

Comment:
You should probably try and extract those constants from angular if possible (as we're now using `import/export` syntax you just have to `import { CONSTANT } from './section.constants'` in the wanted file

Reply:
@marie-j Could be handled as a enhancement. Since this pull request has been already validated I will then process to a merge.

Thanks for your feedback.